Package.swift

```diff
@@ -0,0 +1,38 @@
+// swift-tools-version:5.3
```

Comment:
What is the purpose of this `swift` file?

Reply:
This adds swift bindings, similar to how we have Rust and Node bindings (it is a common addition in many tree-sitter core repos)